checkov/arm/graph_manager.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+import os
+from typing import TYPE_CHECKING, Any
+
+from checkov.arm.graph_builder.local_graph import ArmLocalGraph
+from checkov.arm.utils import get_scannable_file_paths, get_files_definitions
+from checkov.common.graph.graph_builder.consts import GraphSource
+from checkov.common.graph.graph_manager import GraphManager
+
+if TYPE_CHECKING:
+    from checkov.common.typing import LibraryGraphConnector
+
+
+class ArmGraphManager(GraphManager[ArmLocalGraph, "dict[str, dict[str, Any]]"]):
+    def __init__(self, db_connector: LibraryGraphConnector, source: str = GraphSource.ARM) -> None:
+        super().__init__(db_connector=db_connector, parser=None, source=source)
+
+    def build_graph_from_source_directory(
+        self,
+        source_dir: str,
+        local_graph_class: type[ArmLocalGraph] = ArmLocalGraph,
+        render_variables: bool = True,
```

Comment:
shouldn't it be false in default like in others methods?

Reply:
yeah, sure. it is not used anyway 😄